grpc/cubic.proto

```diff
@@ -60,6 +60,24 @@ service Cubic {
     };
   }
 
+  // Compiles recognition context information, such as list of words or phrases
+  // that may appear in the recognition output, into a compact, fast to access
+  // form for a Cubic model. The compiled object may be sent with subsequent
+  // `Recognize` or `StreamingRecognize` requests to aid speech recognition.
+  // This is useful when the context information is constant for multiple
+  // recognize requests. Precompiling context information like this ensures that
+  // there is no added latency to recognition. 
+  //
+  // It is important to note that in order to compile context for a model, that
+  // model has to support context in the first place, which can be verified by
+  // checking its ModelAttributes obtained via the `ListModels` method. Also,
+  // the compiled data will be model specific, that is the data compiled for one
+  // model will generally not be usable with a different model. 
```

Comment:
Annoyingly, a blank line in the comment ends up breaking the generated table in the rendered markdown.  Not sure if the two `<br/>`s will get the visible spacing we want, but it's worth a try.

```suggestion
  // Compiles recognition context information, such as a specialized list of words 
  // or phrases, into a compact, efficient form to send with subsequent
  // `Recognize` or `StreamingRecognize` requests to customize speech recognition.
  // For example, a list of contact names may be compiled in a mobile app and sent 
  // with each recognition request so that the app user's contact names are more likely
  // to be recognized than arbitrary names. This pre-compilation ensures that
  // there is no added latency for the recognition request. 
  // <br/><br/>
  // It is important to note that in order to compile context for a model, that
  // model has to support context in the first place, which can be verified by
  // checking its ModelAttributes obtained via the `ListModels` method. Also,
  // the compiled data will be model specific; that is, the data compiled for one
  // model will generally not be usable with a different model. 
```